rails_startup.bash

```diff
@@ -40,15 +40,17 @@ elif [[ $PASSENGER_APP_ENV = "development" ]]; then
        sudo -E -u app -H yarn install --force
     fi
 fi
+
 echo "*** CREATING CRON ENV FILES ***"
-echo "export PROD_DATABASE_PASSWORD='$PROD_DATABASE_PASSWORD'" >| /home/app/.cron_env
-echo "export SENDGRID_USERNAME='$SENDGRID_USERNAME'" >> /home/app/.cron_env
-echo "export SENDGRID_PASSWORD='$SENDGRID_PASSWORD'" >> /home/app/.cron_env
-echo "export MONGO_LOCALHOST='$MONGO_LOCALHOST'" >> /home/app/.cron_env
-echo "export MONGO_INTERNAL_IP='$MONGO_INTERNAL_IP'" >> /home/app/.cron_env
-echo "export SECRET_KEY_BASE='$SECRET_KEY_BASE'" >> /home/app/.cron_env
-echo "export GOOGLE_CLOUD_PROJECT='$GOOGLE_CLOUD_PROJECT'" >> /home/app/.cron_env
-echo "export MIXPANEL_SECRET='$MIXPANEL_SECRET'" >> /home/app/.cron_env
+echo "# cron environment setup" >| /home/app/.cron_env
+for ENV_VAR in GA_TRACKING_ID GCP_NETWORK_NAME GCP_SUB_NETWORK_NAME LOGSTASH_HOST MIXPANEL_SECRET MONGODB_ADMIN_PASSWORD \
```

Comment:
Non-blocking: While this works fine for the scope of this PR, I think this is going to lead to the exact same issue as before - we'll add an environment variable, then forget to add it here, and `cron` will be missing required data.

What might be more robust (though totally out of scope as it would be a large refactoring) would be to prepend all environment variables with `SCP_`.  That way, we could simply have a regular expression to match all SCP-related secrets from the environment, and then automatically add them to `.cron.env`.

Reply:
yeah, I like that idea -- we should keep that in mind